.github/workflows/publish.yml

```diff
@@ -34,3 +34,22 @@ jobs:
       - name: Deploy TUF-on-CI repository to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@7a9bd943aa5e5175aeb8502edcc6c1c02d398e10 # v4.0.2
+
+  test:
```

Comment:
nit but a more descriptive name for the job would be nice:
* verify-publish
* test-published-repo

or something like that

Reply:
Cheers, I used `test-deployed-repository` (as "deploy" is the name of the previous job